### PR TITLE
(feat): add simple airflow example

### PR DESCRIPTION
**Pull request by sjmignot**

### CODE REVIEW

File: 13_airflow/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    client.trigger_resolver_run(
+        "resolver.get_email_domain"
+    )
+```
+
+## Isolated Python Environment
+
+To isolate the chalkpy dependency from your python environment, you can use airflow's `@task.virtualenv` decorator. Note,
+this slightly slower since a python virtual environment is created for the task.
```
Comment (melrchen) [inline 13_airflow/README.md:48]: nit: * this is

File: 13_airflow/airflow.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    @task.virtualenv(
+        task_id="virtualenv_python", requirements=["chalkpy"], system_site_packages=False
+    )
+    def run_chalk_resolver_virtual_env():
+        """
+        Trigger the resolver.get_email_domain resolver in a virtual environment
+        """
+        from chalk.client import ChalkClient
+
+        # This assumes that CHALK_CLIENT_SECRET & CHALK_CLIENT_ID environment variables
```
Comment (emarx) [inline 13_airflow/airflow.py:40]: let's also ask for the "chalk environment" env var too

File: 13_airflow/airflow.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            "get_users" # this is the name of our sql file resolver {name}.chalk.sql
+        )
+
+    @task
+    def run_chalk_resolver():
+        """
+        Trigger the resolver.get_email_domain resolver
+        """
+
+        # This assumes that CHALK_CLIENT_SECRET & CHALK_CLIENT_ID environment variables
```
Comment (melrchen) [inline 13_airflow/airflow.py:53]: one more here for CHALK_ENVIRONMENT

**Review by melrchen — APPROVED** [review]
⚡

**Pull request merged**